### DISCUSSION TITLE
Weaken dependency constraints II

### DISCUSSION THREAD

**Discussion by pathob** [make-all/tuya-local#1999] [General]
Hi @make-all,

I've put quite some time in writing https://github.com/make-all/tuya-local/discussions/1998, and just closing it without allowing any further discussion is quite a slap in the face.

If you would allow so, I would like to explain to you what NixOS is, why having reproducible builds is a thing, and why I had a point in my original discussion...

**Comment by pathob** [comment]
Sorry, I didn't see I can still comment in the old thread, even though it's closed. I'll close this one and continue the discussion in the old thread.

**Comment by make-all** [comment]
I closed it, because I don't have the bandwidth to support another third party platform in competition with Home Assistant itself.

**Reply by pathob** [reply]
NixOS is not a "platform in competition with Home Assistant itself" but a Linux distribution that, due to its declarative nature and it's guaranteed reproducibility, forms a really nice foundation for a Home Assistant installation. It's no surprise that there are a couple of well-known Home Assistant contributors / maintainers that contribute to / maintain NixOS as well.
